pkgs/applications/misc/citations/default.nix

```diff
@@ -22,21 +22,18 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "citations";
-  version = "0.5.2";
+  version = "0.6.2";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World";
     repo = finalAttrs.pname;
     rev = finalAttrs.version;
-    hash = "sha256-QofsVqulFMiyYKci2vHdQAUJoIIgnPyTRizoBDvYG+g=";
+    hash = "sha256-RV9oQcXzRsNcvZc/8Xt7qZ/88DvHofC2Av0ftxzeF6Q=";
   };
 
   cargoDeps = rustPlatform.importCargoLock {
     lockFile = ./Cargo.lock;
```

Comment:
Then the cargoHash should be used.

Reply:
`stdenv.mkDerivation` does't support such attribute

Reply:
Ah, right. You should use

```nix
  cargoDeps = rustPlatform.fetchCargoTarball {
    inherit src;
    name = "${pname}-${version}";
    hash = "sha256-xxx";
  };
```

Reply:
`importCargoLock `doesn't need a hash, because lock file comes from `src`

Reply:
Because there is no git deps anymore we should use fetchCargoTarball instead and remove the vendored cargo.lock.

Reply:
Still not fixed...